### PR TITLE
Infra.scaffold

### PR DESCRIPTION
**Pull request by jmspring**
Initial work on the `spk infra scaffold` command.

### CODE REVIEW

**Comment by yradsmikham** [comment]
@jmspring let's update the infra README!

File: src/commands/infra/scaffold.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import {
+  disableVerboseLogging,
+  enableVerboseLogging,
+  logger
+} from "../../logger";
+import { generateClusterDefinition, parseVariablesTf } from "./scaffold";
+
+beforeAll(() => {
+  enableVerboseLogging();
+  jest.setTimeout(30000);
```
Comment (yradsmikham) [inline src/commands/infra/scaffold.test.ts]: Not sure if this was carried over from another unit test, but if there's no reason to specify a timeout time, let's not specify it.

**Comment by NathanielRose** [comment]
> @jmspring let's update the infra README!

Updated the README.md

**Review by NathanielRose — APPROVED** [review]
LGTM: Should probably have a `.infra/` folder where all scaffolds will be placed in at the root so its easier to do a gitignore when committing

**Review by yradsmikham — APPROVED** [review]

**Comment by yradsmikham** [comment]
Closes https://github.com/microsoft/bedrock/issues/657.

**Pull request merged**